vignettes/epiprocess.Rmd

```diff
@@ -50,21 +50,23 @@ A data frame or tibble that has `geo_value` and `time_value` columns can be
 converted into an `epi_df` object, using the function `as_epi_df()`. As an
 example, we'll work with daily cumulative COVID-19 cases from four U.S. states:
 CA, FL, NY, and TX, over time span from mid 2020 to early 2022, and we'll use
-the [`covidcast`](https://cmu-delphi.github.io/covidcast/covidcastR/) package to
+the [`delphi-epidata-r`](https://github.com/cmu-delphi/delphi-epidata-r) package to
```

Comment:
Please make sure this is <= 80 characters (I think you can just move "to" onto the next line).